app/models/portfolio_item.rb

```diff
@@ -18,4 +18,12 @@ class PortfolioItem < ApplicationRecord
 
   belongs_to :portfolio
   validates :service_offering_ref, :presence => true
+
+  def resolved_workflow_ref
```

Comment:
Is there a need for a singular version of this method? Since it's an instance method I would think `workflow_ref` would be the result. Unless I am missing something from the requirements.

Reply:
We were thinking for this method that this gives a way to traverse the entire tree (in case the workflow_ref on the item is nil that way it'll go look on the portfolio etc), that way we can invoke this specific workflow. Or is this something that really won't be used except by the method that looks at both the item workflow as well as the platform workflow? @gmcculloug would like your thoughts